Formula/teaxyz.rb

```diff
@@ -0,0 +1,38 @@
+class Teaxyz < Formula
+  desc "Unified package manager"
+  homepage "https://tea.xyz"
+  url "https://github.com/teaxyz/cli/releases/download/v0.13.6/tea-0.13.6.tar.xz"
+  sha256 "88ee0e93a49e2bea80cf75d971bf8fc236c024a42727dafbaf5f7dee4f503d8e"
+  license "Apache-2.0"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "deno" => :build
+
+  conflicts_with "tea", because: "both install `tea` binaries"
+
+  def install
+    system "deno", "compile",
+      "--allow-read",
+      "--allow-write",
+      "--allow-net",
+      "--allow-run",
+      "--allow-env",
+      "--unstable",
+      "--import-map=import-map.json",
+      "--output", bin/"tea",
+      "src/app.ts"
+  end
+
+  def post_install
+    system "#{bin}/tea", "--sync"
+  end
+
+  test do
+    system "#{bin}/tea", "--sync"
```

Comment:
Since testing happens after the install, is this still needed?

Reply:
yes because `$HOME` is modified in the test runner and `$HOME` determines `$TEA_PREFIX`. We could set `$TEA_PREFIX` for the duration of the test to prevent this… if that's possible?

Reply:
In that case it's fine. Lets just leave it like this.